examples/invoice-parser/config.yaml

```diff
@@ -0,0 +1,39 @@
+data_dir: data
+environment_variables: {}
+examples_filename: examples.yaml
+input_type: Any
+model_class_filename: model.py
+model_class_name: InvoiceParserModel
+model_framework: custom
+model_metadata: {}
+model_module_dir: model
+model_name: null
+model_type: custom
+python_version: py39
+requirements: 
+  - cachetools==4.2.2
+  - certifi==2020.12.5
+  - chardet==4.0.0
+  - google-api-core==1.28.0
+  - google-auth==1.30.1
+  - google-cloud-documentai==0.4.0
+  - googleapis-common-protos==1.53.0
+  - grpcio==1.38.0
+  - idna==2.10
+  - packaging==20.9
+  - proto-plus==1.18.1
+  - protobuf==3.17.1
+  - pyasn1==0.4.8
+  - pyasn1-modules==0.2.8
+  - pyparsing==2.4.7
+  - pytz==2021.1
+  - requests==2.25.1
+  - rsa==4.7.2
+  - six==1.16.0
+  - urllib3==1.26.5
+resources:
+  cpu: 500m
+  memory: 512m
+  use_gpu: false
+secrets:
```

Comment:
The secrets that the model needs have to declared here explicitly, otherwise they will not be accessible to the model. I believe we filter for the secrets requested by the model. Listing secrets here is good for documentation as well.